src/controllers/index.js

```diff
@@ -20,4 +20,7 @@ router.get('/sign_up', signUp.get); // router sign_up Page (git)
 router.get('/sign_in', signIn.get); // router sign_in Page (git)
 router.post('/sign_in', signIn.post); // router sign_in Page (post)
 
+router.use(error.client);
```

Comment:
client error handling should be done when a request to any endpoint not listed above it is fired.